src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -347,6 +348,19 @@ public function myCartShouldHaveItems(int $quantity, ProductInterface $product):
         Assert::true($this->hasItemWithNameAndQuantity($response, $product->getName(), $quantity));
     }
 
+    /**
+     * @Then /^my cart shipping total should be "(?:€|£|\$)([^"]+)"$/
+     * @Then I should not see shipping total for my cart
+     */
+    public function myCartShippingFeeShouldBe(string $shippingTotal = '0'): void
+    {
+        $response = $this->cartsClient->getLastResponse();
+
+        Assert::same(
+            $this->responseChecker->getValue($response, 'shippingTotal'),
+            (int) str_replace('.','',$shippingTotal));
```

Comment:
you propably can use in suit `sylius.behat.context.transform.lexical` which transform string with a price to `int`